epimodel/exports/epidemics_org.py

```diff
@@ -103,27 +114,32 @@ def extract_smallish_data(
         hopkins: Optional[pd.DataFrame],
         foretold: Optional[pd.DataFrame],
         timezones: Optional[pd.DataFrame],
+        un_age_dist: Optional[pd.DataFrame],
     ) -> Dict[str, Dict[str, Any]]:
-        d = {
-            "Rates": rates.replace({np.nan: None}).to_dict()
-            if rates is not None
-            else None,
-            "JohnsHopkins": {
+        d = {}
+
+        if rates is not None:
+            d["Rates"] = rates.replace({np.nan: None}).to_dict()
+
+        if hopkins is not None:
+            d["JohnsHopkins"] = {
                 "Date": [x.date().isoformat() for x in hopkins.index],
                 **hopkins.replace({np.nan: None}).to_dict(orient="list"),
             }
-            if hopkins is not None
-            else None,
-            "Foretold": {
+
+        if foretold is not None:
+            d["Foretold"] = {
                 "Date": [x.isoformat() for x in foretold.index],
                 **foretold.replace({np.nan: None})
                 .loc[:, ["Mean", "Variance", "0.05", "0.50", "0.95"]]
                 .to_dict(orient="list"),
             }
-            if foretold is not None
-            else None,
-            "Timezones": timezones["Timezone"].tolist(),
-        }
+
+        d["Timezones"] = timezones["Timezone"].tolist()
+
+        if un_age_dist is not None:
+            d["AgeDist"] = un_age_dist.iloc[:-3].to_dict()
```

Comment:
If I am looking correctly [at this](https://github.com/epidemics/epimodel-covid-data/blob/master/various/age_dist_un.csv), it means all columns but  `95-99 | 100+` ? So the population is not enough, you want to have it by age for future use on frontend for various purposes?

Also, please use explicit column names, pandas is clever and can do e.g. `un_age_dist.loc[:"85-89"]`. Fighting bugs with number-indexed columns is crazy.

Reply:
Yeah that was just the quickest thing I could come up with, you probably know a better way: I just wanted to get rid of the columns "Type","Region Name","Parent Code M49".

Reply:
Btw. I think I could get the true population as we had it in the past under 30 minutes for all countries we export.

Reply:
Sorry I was unclear. What you wrote above was not my intent but rather I wanted to remove just the columns `"Type","Region Name","Parent Code M49"`.

Reply:
Yes I think having the age distribution might be nice because we could use it in the future the calculate the CFR in some other way as those in `rates.csv`.

Reply:
> Sorry I was unclear. What you wrote above was not my intent but rather I wanted to remove just the columns "Type","Region Name","Parent Code M49".

Then do `un_age_dist.drop(columns=["Type","Region Name","Parent"])` to be explicit.


> Yes I think having the age distribution might be nice because we could use it in the future the calculate the CFR in some other way as those in rates.csv.
OK, so keeping as is if you are fine with that?